topics/homework/week2/task_1.py

```diff
@@ -0,0 +1,66 @@
+#Bubble Sort Algorithm
+#In ascending order (regular one)
+user_input = input("Введите последовательность чисел, разделенных пробелами: ")
+numbers = [int(x) for x in user_input.split()]
```

Comment:
Not protected from entered char (or string) like 1 7 x 3

Reply:
Check is added to input_data